.github/workflows/release.yml

```diff
@@ -0,0 +1,204 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        required: true
+        type: string
+      # We use SemVer, anything before v1.0.0 is a pre-release, but this could also include versions like v1.1.0-beta.
+      prerelease:
+        description: 'Prerelease'
+        required: true
+        default: true
+        type: boolean
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+    # Special request from @kyle-rader, so no one can accidentally omit a 'v'.

```

Comment:
lol this is actually part of the comment,

Reply:
Sure is.